gempython/tools/amc_user_functions_uhal.py

```diff
@@ -172,3 +172,10 @@ def blockL1A(device):
 
 def enableL1A(device):
     writeRegister(device, "GEM_AMC.TTC.CTRL.L1A_ENABLE", 0x1)
+
+def getAMCL1Acount(device):
+    return readRegister(device, "GEM_AMC.TTC.L1A_ID")
```

Comment:
ok, I'm going to change it to your suggestion. The goal is just to know how many L1As have been seen by CTP7